package.json

```diff
@@ -22,7 +22,7 @@
     "url": "git+https://github.com/digicatapult/dscp-hyproof-api.git"
   },
   "engines": {
-    "node": "18.x.x",
+    "node": "20.x.x",
```

Comment:
If you are going to do this, can you bump the workflows and dockerfile?

Reply:
dockerfile is already on latest lts (so this). That's actually how I spotted this. Just updated the workflows